docs/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "wikit-docs",
+  "version": "0.0.0",
+  "description": "Storybook for illustrating design tokens and components",
+  "keywords": [
+    "wikit",
+    "docs",
+    "storybook",
+    "tokens",
+    "components"
+  ],
+  "author": "The Wikidata team",
+  "homepage": "https://github.com/wmde/wikit#readme",
+  "license": "BSD-3-Clause",
+  "main": "src/index.js",
+  "files": [
+    "src"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/wmde/wikit.git"
+  },
+  "scripts": {
+    "storybook": "start-storybook"
```

Comment:
Done(ish)

Reply:
We discussed this in the chat. A storybook service was added to docker-compose but, in contrast to most of our projects, we let it pick an available port instead of forcing one (through environment variables).

This has the benefit of not having to define a standard port with the potential of collision on some developers' machines, less boilerplate configuration, and an identical setup for docker and tin users alike.
As the container port does not get mapped to the host system, one needs to open the link to the respective port on the container to see the storybook, but those URLs nicely get presented in the log for the user to click.